entrypoint.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh -l
 set -euo pipefail
 
+source=${SOURCE:-.}
```

Comment:
Would you mind uppercasing this `source` variable for consistency?

Reply:
No, I changed!